### PR TITLE
chore: adds meeting notes for 2019-01-08 meeting

### PR DESCRIPTION
**Pull request by pgte**

### CODE REVIEW

File: meeting-notes/20190108.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+        - Talk to Jim Pick
+
+ -------------
+
+## Notes:
+- Jim demoed a peer-base RGA CRDT working in a terminal using nodejs. XState was used as state machine to change between starting, sending, saved, receiving, etc..
+- Pedro mentioned we must rename peer-base because the owner of the peerbase name declined to give it to us. He will make a pool with a few suggestions so that we can decide on the new name.
+- Pedro showed a PR about the WG roadmap in which we all participated in response to Dirk.
+- Arkady tried to reach a 3rd party to adopt our tech but it's too soon. We need to make the stack more stable and reliable first.
+- Adin and Pedro mentioned that we should learn from what textile and bring that knowledge to peer-base. Later on, and if textile agrees, they can replace parts that they built with parts from peer-base.
```
Comment (daviddias) [inline meeting-notes/20190108.md:174]: Or the other way around :)

File: meeting-notes/20190108.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+        - Event dapp
+    - Get clarity on what's going to happen with Porto ProtoSchool
+        - Talk to Teri
+        - Talk to Jim Pick
+
+ -------------
+
+## Notes:
+- Jim demoed a peer-base RGA CRDT working in a terminal using nodejs. XState was used as state machine to change between starting, sending, saved, receiving, etc..
+- Pedro mentioned we must rename peer-base because the owner of the peerbase name declined to give it to us. He will make a pool with a few suggestions so that we can decide on the new name.
```
Comment (daviddias) [inline meeting-notes/20190108.md:171]: I do not understand why does the responsibility fall on top of a non collaborator to pick the right name.

Comment (satazor) [reply meeting-notes/20190108.md:171]: “He” still refers to Pedro Teixeira, perhaps we should make that clear.

File: meeting-notes/20190108.md (hunk 1 above)
Comment (satazor) [reply meeting-notes/20190108.md:174]: True

File: meeting-notes/20190108.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+- Pedro showed a PR about the WG roadmap in which we all participated in response to Dirk.
+- Arkady tried to reach a 3rd party to adopt our tech but it's too soon. We need to make the stack more stable and reliable first.
+- Adin and Pedro mentioned that we should learn from what textile and bring that knowledge to peer-base. Later on, and if textile agrees, they can replace parts that they built with parts from peer-base.
+
```
Comment (daviddias) [inline meeting-notes/20190108.md:175]: A few questions:

- Shouldn't all the peer-* apps and building blocks move to the peer-base org? Namely:
  - https://github.com/ipfs-shipyard/peer-pad
  - https://github.com/ipfs-shipyard/workshop-todo-dapp
  - https://github.com/ipfs-shipyard/js-delta-crdts
  - https://github.com/ipfs-shipyard/discussify-styleguide
  - https://github.com/ipfs-shipyard/discussify-browser-extension 
  - https://github.com/ipfs-shipyard/js-ipfs-versidag
  - https://github.com/ipfs-shipyard/discussify-styleguide
  - https://github.com/ipfs-shipyard/js-versidag
  - https://github.com/ipfs-shipyard/pm-discussify
  - https://github.com/ipfs-shipyard/peer-star-infra
  - https://github.com/ipfs-shipyard/pm-peer-pad
  - https://github.com/ipfs-shipyard/pm-idm
  - https://github.com/ipfs-shipyard/peer-star
  - https://github.com/ipfs-shipyard/peer-star-react
  - https://github.com/ipfs-shipyard/peer-star-network-vis-react
  - https://github.com/ipfs-shipyard/peer-star-network-vis
  - https://github.com/ipfs-shipyard/peer-star-peer-color
  - https://github.com/ipfs-shipyard/peer-flipchart
  - https://github.com/ipfs-shipyard/demo-ipfs-todo
  - https://github.com/ipfs-shipyard/peerpad-peer-crdt
  - https://github.com/ipfs-shipyard/dapp-identity-api
  - https://github.com/ipfs-shipyard/dapp-identity-api
  - https://github.com/ipfs-shipyard/peer-pad-core
  - https://github.com/ipfs-shipyard/y-ipfs-connector
  - https://github.com/ipfs-shipyard/peer-crdt-textarea-binding
  - https://github.com/ipfs-shipyard/kipster
  - https://github.com/ipfs-shipyard/peer-identity-demo
  - https://github.com/ipfs-shipyard/peer-identity
  - https://github.com/ipfs-shipyard/peer-crdt
  - https://github.com/ipfs-shipyard/peer-crdt-ipfs
  - https://github.com/ipfs-shipyard/peer-blog
  - https://github.com/ipfs-shipyard/peer-cacl
  - https://github.com/ipfs-shipyard/peer-crdt-example
  - https://github.com/ipfs-shipyard/peer-crdt-bind-codemirror
  - https://github.com/ipfs-shipyard/peer-crdt-platform
  - https://github.com/ipfs-shipyard/ipfs-pubsub-peer-monitor
  - https://github.com/ipfs-shipyard/ipfs-live-db
  - https://github.com/ipfs-shipyard/ipfs-iiif-db
  - https://github.com/ipfs-shipyard/tevere
- Any work planned to rebase the current apps on top of peer-base??

File: meeting-notes/20190108.md (hunk 2 above)
Comment (daviddias) [reply meeting-notes/20190108.md:171]: Meanwhile, you could continue using the name `peer-star` which precedes `peer-base`

File: meeting-notes/20190108.md (hunk 3 above)
Comment (daviddias) [reply meeting-notes/20190108.md:175]: Btw, looking at all this repos. If feels it is time to clean up, deprecate and create an onboarding story for all this work. I mean, if I were to build a DApp using IPFS, CRDTs and CACLs, where do I start?

File: meeting-notes/20190108.md (hunk 2 above)
Comment (daviddias) [reply meeting-notes/20190108.md:171]: Other ideas that come to mind:

- peer-platform
- peer-radix

File: meeting-notes/20190108.md (hunk 1 above)
Comment (pgte) [reply meeting-notes/20190108.md:174]: I don't necessarily agree that they're going to use parts of peer-base. They're more advanced than peer-base in many aspects. Right now we're focusing on a) learning what they did and b) their difficulties and how we can help.

File: meeting-notes/20190108.md (hunk 3 above)
Comment (pgte) [reply meeting-notes/20190108.md:175]: Good call, scheduling that in: https://github.com/ipfs/dynamic-data-and-capabilities/issues/64

**Pull request merged**